src/review/Review.tsx

```diff
@@ -17,21 +17,38 @@ export interface ReviewProps {
 const Review = (props: ReviewProps) => {
   const { className, title, text, formattedDatetime, isoDatetime, isResponse = false } = props
 
+  // For responses, the title is not a rating but the name of the peron replying.
+  const titleMicroData = isResponse
+    ? null
+    : {
+        itemScope: true,
+        itemProp: 'reviewRating',
+        itemType: 'https://schema.org/Rating',
+      }
+
+  const titleMicroDataContent = isResponse
+    ? null
+    : {
+        itemProp: 'reviewRating',
+      }
+
   return (
-    <div
+    <blockquote
+      itemScope
+      itemType="http://schema.org/Review"
       className={cc([
         className,
         {
           'kirk-is-review-response': isResponse,
         },
       ])}
     >
-      <h2 className="kirk-review-title">
-        <TextTitle>{title}</TextTitle>
+      <h2 className="kirk-review-title" {...titleMicroData}>
```

Comment:
`.kirk-review-title` redefine the `font-size`. Instead it should take care, mostly of spacing.
Should we define a _rule_ on what we can redefine form `TextComponents` (colors, font-size, line-height) otherwise it will be another `Text` component.

Another tip here is redefining this component with "Styled Components" instead of "classes".

Reply:
It's the normalization of the font-size, not defining a new size.
We had a discussion offliine with Jerome (sorry I should have brought you in too :/) and there is a big overhaul to do about titles. In particular, we should not use <Title> anymore which is just poop. But we decided to not take care of this right now, at least in this PR.